src/lib/Components/Buttons/CaretButton.tsx

```diff
@@ -0,0 +1,19 @@
+import { Display, Flex } from "@artsy/palette"
+import React from "react"
+import { GestureResponderEvent, Image, TouchableWithoutFeedback } from "react-native"
```

Comment:
FYI We have a custom image component (`OpaqueImageView`) which is built on top of our native image cache.
It's necessary to use here but just so you know